ts/Extensions/ExportData.ts

```diff
@@ -469,6 +471,71 @@ addEvent(Chart, 'render', function (): void {
     }
 });
 
+addEvent(Chart, 'afterViewData', function (): void {
+    const chart = this,
+        dataTableDiv = chart.dataTableDiv,
+        row = document.getElementsByClassName('highcharts-table-sort-row')[0],
```

Comment:
Do we need the `highcharts-table-sort-row` classname? The top row should be selectable as `thead tr`, while the other rows are not inside a `thead`.